libmscore/property.h

```diff
@@ -240,6 +242,7 @@ enum class P_TYPE : char {
       COLOR,
       DIRECTION,      // enum class MScore::Direction
       DIRECTION_H,    // enum class MScore::DirectionH
+      ORNAMENT_STYLE, // enum class MScore::OrnamentStyle
```

Comment:
What do you mean?  Once is P_TYPE::ORNAMENT_STYLE and the other is P_ID::ORNAMENT_STYLE

Is that what you mean?


> On 05 May 2015, at 11:41, Nicolas Froment <notifications@github.com> wrote:
> 
> In libmscore/property.h <https://github.com/musescore/MuseScore/pull/1946#discussion_r29654920>:
> 
> > @@ -240,6 +242,7 @@ enum class P_TYPE : char {
> >        COLOR,
> >        DIRECTION,      // enum class MScore::Direction
> >        DIRECTION_H,    // enum class MScore::DirectionH
> > +      ORNAMENT_STYLE, // enum class MScore::OrnamentStyle
> Why twice ORNAMENT_STYLE ?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/musescore/MuseScore/pull/1946/files#r29654920>.
>